tests/foreman/ui_airgun/test_partitiontable.py

```diff
@@ -0,0 +1,211 @@
+    template = gen_string('alpha')
+    default_org = entities.Organization().search(
+        query={'search': 'name="{0}"'.format(DEFAULT_ORG)})[0]
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': True,
+            'organizations.assigned': [default_org.name],
+            'template': template
+        })
+        session.organization.select(DEFAULT_ORG)
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_non_default_for_organization(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    organizations = entities.Organization().create()
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': False,
+            'organizations.assigned': [organizations.name],
+            'template': template
+        })
+        session.organization.select(org_name=organizations.name)
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_default_for_location(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    default_loc = entities.Location().search(
+        query={'search': 'name="{0}"'.format(DEFAULT_LOC)})[0]
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': True,
+            'locations.assigned': [default_loc.name],
+            'template': template
+        })
+        session.location.select(DEFAULT_LOC)
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_create_non_default_for_location(session):
+    name = gen_string('alpha')
+    template = gen_string('alpha')
+    locations = entities.Location().create()
+    with session:
+        session.partitiontable.create({
+            'name': name,
+            'default': False,
+            'locations.assigned': [locations.name],
+            'template': template
+        })
+        session.location.select(loc_name=locations.name)
+        assert session.partitiontable.search(name)[0]['Name'] == name
+
+
+def test_positive_update(session):
+    old_name = gen_string('alpha')
+    new_name = gen_string('alpha')
+    template = gen_string('alpha')
+    template_new = gen_string('alpha')
+    os_family_selection = {'os_family': 'Red Hat'}
+    new_os_fs = {'os_family': 'Debian'}
+    with session:
+        session.partitiontable.create({
+            'name': old_name,
+            'template': template,
+            'os_family_selection': os_family_selection,
+        })
+        session.partitiontable.update({
+            'name': new_name,
+            'os_family_selection': new_os_fs,
+            'template': template_new,
+        }, old_name)
+        pt = session.partitiontable.read(new_name)
+        assert pt['os_family_selection']['os_family'] == new_os_fs['os_family']
+        assert pt['name'] == new_name
```

Comment:
Is it possible to assert template changed here?

Reply:
with ```read``` template return None, if you know how to read information from ACEEditor,  it will help me, for now I am doing the asserts without template.